cmd/status.go

```diff
@@ -15,13 +15,28 @@ import (
 func NewStatusCommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
 	o := &StatusOptions{IOStreams: ioStreams}
 	cmd := &cobra.Command{
-		Use:     "status",
-		Short:   "Show status of working directory",
-		Long:    ``,
-		Example: ``,
+		Use:   "status [DATASET]",
+		Short: "Show status of working directory",
+		Long: `List what components and files of the working directory's dataset have been
+added, removed, or changed.
+
+If you specify a DATASET, this will list what components were changed in a
+particular commit (or the latest commit if none is specified). You can specify
+a commit alongside a dataset like:
+
+    me/dataset_name@/ipfs/Qmu...`,
```

Comment:
I had no idea you could do this at all. 😯

Reply:
(FWIW, seems kinda odd for the `status` command, but a useful feature to have overall.)

Reply:
FYI, I'm going to be trying to remove the ability to use an explicit ref with path for the `status` command. We use this functionality in Desktop in order to see what changed in previous commits. I think it makes more sense as a separate command `showcommit`, so want to extract it out. These changes are fine for now, just answering your curiosity.

Reply:
👍 Was thinking it fit better in log or something similar. Makes sense.